### DISCUSSION TITLE
How to get prove transaction with withdraw transaction from L2

### DISCUSSION THREAD

**Discussion by frongrider2** [ethereum-optimism/developers#573] [Application Development]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

How to get prove transaction with withdraw transaction from L2?

I want prove and finalize transaction from withdrawal flow

**Comment by imtipi** [comment]
do you mean by manual or by code
manual:https://app.optimism.io/bridge/deposit
code:https://viem.sh/op-stack/actions/proveWithdrawal
https://viem.sh/op-stack/actions/finalizeWithdrawal
